VideoLocker/src/main/java/org/edx/mobile/util/FileUtil.java

```diff
@@ -38,4 +41,25 @@
             inputStream.close();
         }
     }
+
+    /**
+     * Deletes a single file and/or recursively deletes files/folders within a folder before
+     * deleting it.
```

Comment:
The 'and/or' should technically be just 'or'. Or the description could be simplified to e.g. just "Deletes a file or directory and all contents recursively.".